NachoSpigot-Server/src/main/java/net/minecraft/server/MathHelper.java

```diff
@@ -10,29 +12,32 @@
     private static final double d;
     private static final double[] e;
     private static final double[] f;
+    private static final boolean fastMathMode = NachoConfig.enableFastMath;
 
     public static float sin(float var0) {
-        return b[(int)(var0 * 10430.378F) & '\uffff'];
+        return (fastMathMode == true ? ((float) FastMath.sin(var0)) : (b[(int)(var0 * 10430.378F) & '\uffff']));
```

Comment:
```suggestion
        return b[(int)(var0 * 10430.378F) & '\uffff'];
```

This not only would cause anti-cheat issues but also probably issues with the server. This will also let us remove the caution for Fast math, as these are the only methods that are different in my tests.

Reply:
You should do other tests then.